### DISCUSSION TITLE
Query editor - New Line and Run query keystrokes

### DISCUSSION THREAD

**Discussion by gelicia** [grafana/grafana#52616] [Observability]
A user on twitter brought up an inconsistency between the Explore UI and the PrometheusIO's UI, where explore creates a new line by hitting enter, and shift enter runs the query, and the opposite happens in PrometheusIO. It seems that our editors used to be consistent but that is no longer the case. We would like to not switch this on people who may be used to how it is now, but still give the option for people who may be switching between applications so they can match.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

**Reply by krisztianfekete** [reply]
Oh, that's a pity. I was that user on Twitter, and it seems like lots of people were interested in this one. I'll reopen it.

**Reply by gelicia** [reply]
Thank you!

**Reply by krisztianfekete** [reply]
Here's the issue as a reference: https://github.com/grafana/grafana/issues/82469
